app/models/media_resource.rb

```diff
@@ -46,13 +46,23 @@ def update_from_fixer(fixer_task)
   end
 
   def update_attributes_with_fixer_info(info)
-    self.mime_type = info['content_type']
-    self.file_size = info['size'].to_i
+    # this is not working
+    # self.mime_type = info['content_type']
```

Comment:
This isn't working in the "is null" sense, or in the "doesn't have a sane default" sense?